cli.js

```diff
@@ -358,12 +358,17 @@ process.on('uncaughtException', function(err) {
       var sfxBundle = true;
 
     case 'bundle':
-      options = readOptions(args, ['--inject', '--yes', '--skip-source-maps', '--minify',  '--no-mangle', '--hires-source-maps', '--no-runtime']);
+      options = readOptions(args, ['--inject', '--yes', '--skip-source-maps', '--minify',  '--no-mangle', '--hires-source-maps', '--no-runtime', '--inline-source-maps']);
       if (options.yes)
         ui.useDefaults();
       options.sourceMaps = !options['skip-source-maps'];
       options.lowResSourceMaps = !options['hires-source-maps'];
       options.mangle = !options['no-mangle'];
+      
+      if (options['inline-source-maps']){
+        options.sourceMapContents = 'inline';
```

Comment:
@johnraz Yeah that's the right summary for the behaviours - although I'd remove some ambiguity by saying "embeds the sourcemap in the output" rather :wink: 

In JSPM we want to reserve `inline` to refer to embedding the sourcemap in the output file, and do not want to use this command line option to refer to the weaker "include sources in the source map" action. 

We should add a separate command line option for `sourceMapContents`, like `--source-map-include-contents`.

RE: the exception you're getting, I think https://github.com/systemjs/plugin-css/pull/37 should sort you out, but it has not been tested yet